Distraction Shield/classes/UserSettings.js

```diff
@@ -7,7 +7,7 @@ import * as storage from '../modules/storage/storage'
  * Since the current state is saved here too.
  */
 export default class UserSettings {
-    constructor() {
+    constructor(id = undefined) {
```

Comment:
it's assigning a default parameter, basically if something fucks up with the uuid generator, this wont result in all logs being invalid.